plugins/scaleway/access_key.go

```diff
@@ -0,0 +1,147 @@
+package scaleway
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.AccessKey,
+		DocsURL:       sdk.URL("https://www.scaleway.com/en/docs/iam/how-to/create-api-keys/"),
+		ManagementURL: sdk.URL("https://console.scaleway.com/iam/api-keys"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.AccessKeyID,
+				MarkdownDescription: "Access key",
```

Comment:
```suggestion
				MarkdownDescription: "Access key ID",
```

To prevent confusion with the secret key itself